README.md

```diff
@@ -28,6 +28,72 @@ When upgrading to this version of Debugger for Mainframe from an older version,
 #### Client
 - Java version 8.0 or higher.
 
+### Set up Secure Connection to InterTest Server
+
+#### Prerequisites
+- InterTest server running and configured to support secure communication.
+- Security certificate for the InterTest server.
+
+**Follow these steps**
+
+1. Download the Server Certificate to your local machine
+
+2. Import certificate with entry name HOSTNAME to the trust store, for example:
+
+    *jdk0.0.0_000\abc\Lib\security* named cacerts of the JRE under which the Debugger for Mainframe DA client is running.
```

Comment:
This step is grammatically incorrect, and not very clearly written. What does "named cacerts" mean on the second line? What is that path an example of and is it really necessary?
Can't we just say "import the certificate to the trust store of the JRE under which the Debugger for Mainframe DA client is running. Use the entry name HOSTNAME."